mixin/dashboards/compact.libsonnet

```diff
@@ -18,7 +18,7 @@ local g = import '../lib/thanos-grafana-builder/builder.libsonnet';
             'Shows rate of execution for compactions against blocks that are stored in the bucket by compaction group.'
           ) +
           g.queryPanel(
-            'sum(rate(thanos_compact_group_compactions_total{namespace="$namespace",job=~"$job"}[$interval])) by (job, group)',
+            'sum by (job, group) (rate(thanos_compact_group_compactions_total{%s}[$interval]))' % selector,
```

Comment:
I think we might want to put `(%(aggregators)s)`. The reason is that by removing namespace we allow multiple namespaces (and clusters - which was before) to be part of this aggregation. So suddenly we are returning sum-med number of group compactions, not one. And it's hidden to the dashboard user, I know it's a tedious work, but it sounds crucial to me 🤔